v_6_0_0/types.go

```diff
@@ -115,6 +117,16 @@ type HyperkubePodHostMount struct {
 	ReadOnly bool
 }
 
+type MultiMastersSpec struct {
+	// If set to true, k8scloduconfig will render master template for cluster of 3 masters.
+	// Defaults to false.
+	Enabled bool
+	// Etcd initial cluster is config which define which etcd are members of the cluster.
+	// The format look like to this: `etcd0=https://10.1.1.1:2380,etcd1=https://10.1.1.2:2380,etcd2=https://10.1.1.3:2380`
+	// Where 10.1.1.1, 10.1.1.2, 10.1.1.3 can be either IP or DNS  of master machine where is etcd listening.
+	EtcdInitialCluster string
```

Comment:
I wanted initially to build this based on `BaseDomain` but then this would complicate things for China as there is no route53, so the operator needs to provide this value-based either on DNS or IP. It should be quite easy to do via some `key` function.

Reply:
Hmm maybe we need to use DNS anyway because of the SNI in SSL certs